README.md

```diff
@@ -53,6 +53,55 @@ There are three steps to the CI workflow:
      doesn't update any services, only manifests, so the push from the `deploy`
      job will never run another `deploy` job.
 
+## WIP: Deploy Slack and Hr service locally
+
+Use the 'docker-desktop' local k8s cluster.
+
+Initialize Dapr on the cluster (https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/)
+
+```bash
+dapr init -k
+```
+
+Create the things. The version of Kustomize inside kubectl is quite old, so we need to use kustomize directly. [Install](https://kubectl.docs.kubernetes.io/installation/kustomize/) with `brew install customize`. This works when using the "docker-desktop" k8s cluster, if you are using minikube you will probably need to [set the docker daemon](https://stackoverflow.com/questions/42564058/how-to-use-local-docker-images-with-minikube)
+
+```bash
+docker build --progress=plain -t slack:latest ./lib/slack
+docker build --progress=plain -t hr:latest ./lib/hr
```

Comment:
Might be worth using `docker-compose`?

Reply:
Hi David, I don't think docker compose is the best option here (although I'm not an expert and am always happy to take on opinions and learn things). As I understand it the sweet spot for docker compose is defining how multiple docker images relate to each other (shared network, disk, ports and suchlike). Here it would turn a two line command in to a one line command, but is another thing to learn / worry about, and requires a docker-compose.yml, which will be more than a line long! Also the two commands here don't have any shared things, so its not in the docker compose sweet spot. It would certainly be worth thinking about as we add more docker images though.

Reply:
Yeah that's a fair argument against it 👍 
Was thinking ahead if we add more services it might be handy to run one command to `up` them all but looks like it's not needed for now.